corehq/apps/users/views/__init__.py

```diff
@@ -1069,6 +1054,27 @@ def post(self, request, *args, **kwargs):
         except WorksheetNotFound:
             self.group_specs = []
 
+
+class UploadWebUsers(BaseUploadUser):
+    template_name = 'hqwebapp/bulk_upload.html'
+    urlname = 'upload_web_users'
+    page_title = ugettext_noop("Bulk Upload Web Users")
+
+    @method_decorator(always_allow_project_access)
```

Comment:
why does this have so many more decorators than the method it replaced?

Reply:
Originally when it was inheriting from `BaseManageWebUserView`, that has the additional decorators [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/views/__init__.py#L917) that I thought would be important to carry over. Can remove though